src/AdaptiveCurveIrm.sol

```diff
@@ -129,12 +91,12 @@ contract AdaptiveCurveIrm is IIrm {
 
         if (startRateAtTarget == 0) {
             // First interaction.
-            avgRateAtTarget = INITIAL_RATE_AT_TARGET;
-            endRateAtTarget = INITIAL_RATE_AT_TARGET;
+            avgRateAtTarget = ConstantsLib.INITIAL_RATE_AT_TARGET;
+            endRateAtTarget = ConstantsLib.INITIAL_RATE_AT_TARGET;
         } else {
-            // Note that the speed is assumed constant between two interactions, but in theory it increases because of
-            // interests. So the rate will be slightly underestimated.
-            int256 speed = ADJUSTMENT_SPEED.wMulDown(err);
+            // The speed is assumed constant between two updates, but it is in fact not constant because of interest.
+            // So the rate will be underestimated.
```

Comment:
This natspec change has been made because of the https://cantina.xyz/code/8409a0ce-6c21-4cc9-8ef2-bd77ce7425af/findings/0b0fd532-796b-4663-b3f5-03c0c8caa701 cantina contest report.

Are you going to only update the natspec without actively handling the edge case?

Also, I suggest considering enhancing the dev comment by giving more details on **when**, **why** and **how much** the `rate` can be **underestimated**.

Reply:
>Are you going to only update the natspec without actively handling the edge case?

yes, as we don't have any satisfying solution for this problem